portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/roleSwitch-snippet.vm

```diff
@@ -8,7 +8,6 @@
     #else
         <a id="roleSwitchDropDownToggle" href="javascript:void('0')" aria-hidden='true' aria-haspopup="true" title='${rloader.rs_menuTooltip}'>
             <i class="fa fa-user-secret" aria-hidden="true"></i>
-            <span class='skip'>${rloader.rs_toggleRoleSwitcher}</span>
```

Comment:
So by removing this helper text, the SR user on a mobile device now depends on the title attribute in the above <a> tag only. And that title says "Show View options". That doesn't seem very helpful.

I would recommend adding this rs_toggleRoleSwitcher to the properties file